Source/CombatExtended/Contrib/ProjectileImpactFX/ImpactFleckThrower.cs

```diff
@@ -148,6 +150,18 @@ public static void ThrowFleck(Vector3 loc, IntVec3 position, Map map, Projectile
                 map.flecks.CreateFleck(creationData);
             }
 
+            if (StuckProjectileFleck != null)
+            {
+                FleckCreationData creationData = FleckMaker.GetDataStatic(loc, map, StuckProjectileFleck);
+                creationData.scale = ext.StuckProjectileFleckSize;
+                creationData.rotation = direction > 0 ? 360 - direction : 0 - direction;
```

Comment:
Modular arithmatic is your friend, as it lets you make this branchless.  The JIT will thank you.

I believe that should be `creationData.rotation = (- direction) % 360;`

Reply:
Oh yeah, thats a lot cleaner. Changed it